README.md

```diff
@@ -21,7 +21,11 @@ Before you can start working with Yari, you need to:
 <!-- markdownlint-disable list-marker-space -->
 
 1.  Install [git](https://git-scm.com/), [Node.js](https://nodejs.org), and
-    [Yarn 1](https://classic.yarnpkg.com/en/docs/install).
+    [Yarn 1](https://classic.yarnpkg.com/en/docs/install). Alternatively, if you
+    have [Nix](https://nixos.org) installed on your system, you can use the dev
+    shell provided in the `flake.nix` file at the root of this repo with
+    `nix develop`, to quickly setup an environment with all of the required
```

Comment:
I my (macOS) environment, to make it work, I needed to run with additional flags:
```
$ nix develop --extra-experimental-features nix-command --extra-experimental-features flakes
```
…otherwise, I got these fatal errors:
> error: experimental Nix feature 'nix-command' is disabled; use '--extra-experimental-features nix-command' to override
>
> error: experimental Nix feature 'flakes' is disabled; use '--extra-experimental-features flakes' to override

Reply:
Good catch - flakes have unfortunately been stuck in "experimental" purgatory for years at this point, and while most Nix users are aware of the flags that need to be set, it's probably still a good idea to mention that in case new users come along.

Reply:
I've linked to the relevant instructions on the [NixOS wiki](https://nixos.wiki/wiki/Flakes#Enable_flakes) for setting up flakes - that should hopefully explain it well enough.